ehsm_kms_service/secret_manager_apis.js

```diff
@@ -165,24 +192,12 @@ const createSecret = async (res, appid, payload, DB) => {
             return
         }
 
-        //Query keyblob through encryptionkeyid, encrypt secretdata through keyblob
-        if (encryptionKeyId != '' && typeof (encryptionKeyId) == 'string') {
-            let key_id_res = await DB.partitionedFind('cmk', queryKeyId(appid, encryptionKeyId))
-            if (key_id_res.docs.length == 0) {
-                res.send(_result(400, 'Cannot find key id'))
-                return
-            } else {
-                keyBlob = key_id_res.docs[0].keyBlob
-            }
-        } else if (encryptionKeyId == "" || encryptionKeyId == undefined) {
-            let default_cmk_res = await DB.partitionedFind('user_info', queryDefaultCMK(appid))
-            if (default_cmk_res.docs.length == 0) {
-                res.send(_result(400, 'Cannot find default CMK'))
-                return
-            }
-            keyBlob = default_cmk_res.docs[0].sm_default_cmk
+        // query cmk and encrypt secretData
+        retCMK = await queryCMK(DB, appid, encryptionKeyId)
```

Comment:
change the function name as get_SM_Masterkey

and better to change the keyBlob -> SM_MasterKey

Reply:
It will be change